apps/partner2/stylesheets/overview.styl

```diff
@@ -28,7 +28,7 @@
       min-height 100px
       .partner-overview-section-title
         avant-garde-size('m-headline')
-    &[data-module=about] + [data-module=news]
+    &[data-module=news]
```

Comment:
Oh... Horizontally - meaning - not stretched horizontally, but in columns horizontally. 

And what happens - when there is 4 or 2 events? Do we do completely dynamic columns for those with min and max width? Or have fixed width and just go to the next line? @starsirius? @brentcas?

And that is what dropping the title was from:)

In any case - merging this current version since it's better than what we have in production now, and we should open new issue for the other formatting (I think)